packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/onboarding/Onboarding.page.tsx

```diff
@@ -15,42 +17,58 @@ const Onboarding = () => {
         <Guides />
       </div>
       <h2>{t('title')}</h2>
-      <img src={''} className="max-h-[250px]" alt="ai notebooks" />
-      <p>{t('description1')}</p>
+      <img
+        src={onboardingImgSrc}
+        className="max-h-[250px]"
+        alt="ai notebooks"
+      />
+      <p className="font-bold">{t('description1')}</p>
       <p className="font-bold">{t('description2')}</p>
       <p>{t('description3')}</p>
       <Button data-testid="create-notebook-link" asChild>
         <Link
-          to="new"
+          to="../new"
           className="hover:no-underline hover:text-primary-foreground"
         >
           {t('createNotebookButton')}
         </Link>
       </Button>
       <div className="flex flex-col md:grid md:grid-cols-3 gap-2">
-        {/* 
         <OnboardingTile
-          title={t('cardGuidestitle')}
-          description={t('cardGuidesDescription')}
-          content={t('cardGuidesContent')}
-          href="https://docs.ovh.com/gb/en/publiccloud/databases/"
+          title={t('cardTutotitle')}
+          description={t('cardTuto1Description')}
+          content={t('cardTuto1Content')}
+          href="https://docs.ovh.com/gb/en/publiccloud/ai/"
```

Comment:
We always have the English links here

Reply:
Yes thats already the case here in production. I said, let's go in prod like this and modify after if needed.